init.lua

```diff
@@ -13,7 +13,7 @@
 
 -- Edited by Jasper den Ouden (a few commits now)
 
-dofile(minetest.get_modpath("painting").."/crafts.lua")
+dofile(minetest.get_modpath(minetest.get_current_modname()).."/crafts.lua")
```

Comment:
This mod needs to be called `painting` because other mods may have `painting` in their dependency list, so you can assume that the current mod name does not change.

Reply:
I prefer automatic, since if you download it on github it is renamed to painting-master...

Reply:
Renaming the folder is mentioned at the official installation instructions: https://wiki.minetest.net/Help:Installing_Mods
The mod probably does not work if it has another name because of how the nodes and similar things are registered.